### PR TITLE
Add Job route for COVID-19 reservation to utah lonepeak

### PR DESCRIPTION
**Pull request by jstidd**

### CODE REVIEW

**Comment by brianhlin** [comment]
Since you're using the same queue and resource requests for COVID-19 jobs vs non-COVID-19 jobs, these changes aren't actually necessary! Basically the OSG factories will send the following types of pilot jobs to your site:

- COVID-19 only pilots (i.e., `IsCOVID19 = True`)
- Generic OSG Pilots, which currently prioritize COVID-19 user jobs

So if you're treating the two types of pilots the same, you're basically leaving COVID-19 prioritization up to the OSG.

**Comment by Mansalu** [comment]
Good to know. Thank you for the clarification.

I am in discussions with folks at CHPC about possibly creating a specialized reservation for COVID-19 work that wouldn't be preemptable. If they choose to go that way we will adjust this PR then. For now I'll mark this as draft.

**Comment by Mansalu** [comment]
I'm trying to verify the viability of `set_reservation = "covid-reservation"` it looks like this might not make it through Bosco to slurm.

We'll hold off on merging this for now.

**Comment by Mansalu** [comment]
Alternatively, we can try to have Brian H create a new partition/QOS within slurm for this purpose.

**Comment by Mansalu** [comment]
I think this is what we need. @brianhlin Can you take a look once you get the chance?

File: SLATE_US_UUTAH_LONEPEAK/lonepeak-slate-values.yaml [hunk 1]
```diff
@@ -42,6 +42,21 @@ Networking:
...
+    Requirements = (Owner == "osguserl");
+    set_default_queue = "lonepeak-osg";
+    Requirements = (TARGET.IsCOVID19 =?= true);
+    # 16 GB
+    set_default_maxMemory = 16384;
+    # 72 hours
+    set_BatchRuntime = 4320;
+    # 4 core jobs
+    set_default_xcount = 4;
+    set_reservation = "covid-reservation"
```
Comment (brianhlin) [inline SLATE_US_UUTAH_LONEPEAK/lonepeak-slate-values.yaml]:
```suggestion
    set_reservation = "covid-reservation";
```

Missing semicolon

**Review by brianhlin — APPROVED** [review]
One minor change but otherwise LGTM

**Comment by Mansalu** [comment]
Thanks Brian!

**Review by Mansalu — APPROVED** [review]
LGTM

**Pull request merged**